### ISSUE TITLE
User gives multiple choice reason for declining swap offer

### ISSUE THREAD

**Issue by tdg** [swapmyvote/swapmyvote#984]
We get quite a bit of feedback saying 'I was declined X number of times' etc and we have no visibility on why this is. 

This would be optional however
we could then pass it on to the offering voter. 

We speculate most declines are for one of a small number of reasons (please add) - we would obviously word these appropriately

- the offered vote is not helpful to (my preferred party)
- there's something fishy about the offering profile 
 (Maybe they haven't verified?)
- I'm looking for an even more helpful swap for (party)

**Comment by baob** [comment]
Instead of deleting the swap (which I'm pretty sure is what we do right now we should either
* keep it and mark it declined
* record it in some other way (noting 2 parties and two constituencies, but possibly not the user

So we can do later analysis.

We should also ask "Why" and offer,  probably, a preselected list of reasons and a text field for "other".

**Comment by aspiers** [comment]
Yes, these are all fantastic ideas.  You're right that deleting it is both what we do and a bad idea.
